.github/workflows/commitlint.yaml

```diff
@@ -0,0 +1,26 @@
+---
+name: Commitlint
+on:
+  push:
+    branches-ignore:
+      - "main"
+
+jobs:
+  commitlint:
+    runs-on: ubuntu-20.04
+    timeout-minutes: 5
+
+    steps:
+      - uses: actions/checkout@v2
+        with:
+          fetch-depth: 50
+      - uses: actions/setup-node@v1
```

Comment:
There is [already v2](https://github.com/actions/setup-node/releases).

Reply:
Great, updated.